dal/db.go

```diff
@@ -11,40 +11,36 @@ import (
 
 // IkaSignRequest represents a row in the `ika_sign_requests` table.
 type IkaSignRequest struct {
-	// ID is a sign request reported and managed by Native
-	ID        uint64    `json:"id"`
-	Payload   Payload   `json:"payload"`
 	DWalletID string    `json:"dwallet_id"`
 	UserSig   string    `json:"user_sig"`
+	Payload   Payload   `json:"payload"`
 	FinalSig  Signature `json:"final_sig"`
+	ID        uint64    `json:"id"` // ID is a sign request reported and managed by Native
 	Timestamp int64     `json:"time"`
 }
 
 // IkaTx represents a row in the `ika_txs` table.
 type IkaTx struct {
-	// SrID is the IkaSignRequest.ID (from Native)
-	SrID      uint64      `json:"sr_id"`
-	Status    IkaTxStatus `json:"status"`
 	IkaTxID   string      `json:"ika_tx_id"`
-	Timestamp int64       `json:"timestamp"`
 	Note      string      `json:"note"`
+	SrID      uint64      `json:"sr_id"` // SrID is the IkaSignRequest.ID (from Native)
+	Timestamp int64       `json:"timestamp"`
```

Comment:
let's put SrID and Timestamp to the top - it should work.

Reply:
wont work